kinesis-produce-consume/src/main/resources/bootstrap.yml

```diff
@@ -0,0 +1,5 @@
+spring:
+  application:
+    name: KinesisProducer
```

Comment:
Looks like this app not only `Producer`.
Can we rename it to better reflect reality?

Also new line in the end of each file is a rule.

Thanks for understanding